### ISSUE TITLE
[lab-4] Issues

### ISSUE THREAD

**Issue by mykola-fant-ztu** [Ajoke32/SoftwareDesignLab#3]
- Weird that the log methods don't accept the message as an argument :) https://github.com/Ajoke32/SoftwareDesignLab/blob/master/lab4/PatternsPartTwo/PatternsPartTwo/Adapter/Logger.cs#L14;
- The actual adapter is here (https://github.com/Ajoke32/SoftwareDesignLab/blob/master/lab4/PatternsPartTwo/PatternsPartTwo/Adapter/FileLogger.cs), but you have created another class, that does NOTHING and called it "Adapter" (https://github.com/Ajoke32/SoftwareDesignLab/blob/master/lab4/PatternsPartTwo/PatternsPartTwo/Adapter/FileLoggerAdapter.cs), hard to follow such logic;

**Comment by Ajoke32** [comment]
I see. Thanks for the review
